PhysicsTools/PatAlgos/python/slimming/isolatedTracks_cfi.py

```diff
@@ -35,6 +44,10 @@
     dEdxDataPixel = cms.InputTag("dedxPixelHarmonic2"),
     dEdxHitInfo = cms.InputTag("dedxHitInfo"),
     dEdxHitInfoPrescale = cms.InputTag("dedxHitInfo","prescale"), 
+    EBRecHits =  cms.InputTag  ("reducedEcalRecHitsEB"),
+    EERecHits =  cms.InputTag  ("reducedEcalRecHitsEE"),
+    HBHERecHits =  cms.InputTag  ("reducedHcalRecHits", "hbhereco"),
```

Comment:
@mariadalfonso  
Indeed...

Actually it's (historically) a mixture of HBHERecHits with non-0 *noise flags* (small part of it) and HBHERecHits
interesting" for Egamma (majority)
https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/HcalRecProducers/python/HcalHitSelection_cfi.py

Reply:
@mariadalfonso @abdoulline for the Disappearing Tracks search (EXO-19-010 most recently), we want to sum *all* calorimeter deposits near to a generalTrack. Even for the most severely pathological hit, a 'disappearing' track candidate with large energy near it would be suspect.

It's really a downside that the AOD recHits collections are slimmed as they are, if I understand the definition of these collections right then a non-noise-flagged, non-"interesting for EGamma" HCAL recHIt is lost. We haven't yet studied these recHit sums at the RECO tier; a PR for that type of change would be far more disruptive than this PR.

@czkaiweb has been asked to discuss this in an xPOG meeting, we're glad to add more context/discussion there since this is a matter of Physics motivations.